### PR TITLE
MISUV-8275 Decommission Feature Switches: IV Uplift, Payment Allocation, Coding Out

### PR DESCRIPTION
**Pull request by dnyuar**

### CODE REVIEW

File: app/controllers/ChargeSummaryController.scala [hunk 1]
```diff
@@ -237,9 +227,9 @@ class ChargeSummaryController @Inject()(val auth: AuthenticatorPredicate,
 
   def mandatoryViewDataPresent(isLatePaymentCharge: Boolean, documentDetailWithDueDate: DocumentDetailWithDueDate)(implicit user: MtdItUser[_]): Either[ErrorCode, Boolean] = {
 
-    val viewSection1 = isEnabled(ChargeHistory) && (!isLatePaymentCharge && !(isEnabled(CodingOut) && documentDetailWithDueDate.documentDetail.isPayeSelfAssessment))
```
Comment (danielmaltz1) [inline app/controllers/ChargeSummaryController.scala:240]: Looks like this one would have only been true when coding out was disabled, so would always be false now

**Comment by avelx** [comment]
@dnyuar - we would have to raise another story to tidy up code: remove not used methods params etc;

**Comment by dnyuar** [comment]
Yep, I'm currently creating a Jira ticket to tidy up and remove the unused code

**Comment by dnyuar** [comment]
Tech debt ticket number is MISUV-8681

**Review by danielmaltz1 — APPROVED** [review]

**Pull request merged**